transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollServerSocketChannel.java

```diff
@@ -53,17 +52,11 @@ public EpollServerSocketChannel(int fd) {
 
     EpollServerSocketChannel(LinuxSocket fd) {
         super(fd);
-        // As we create an EpollServerSocketChannel from a FileDescriptor we should try to obtain the remote and local
-        // address from it. This is needed as the FileDescriptor may be bound already.
-        local = fd.localAddress();
```

Comment:
related to https://github.com/netty/netty/pull/6193#discussion_r124035232

Just wanted to double check if this change in behavior is intended ... if the connection hasn't completed synchronously the local address will now be null. Will this always occur synchronously for server sockets? If not this may be an unexpected change.